pom.xml

```diff
@@ -226,9 +227,31 @@
 		</dependency>
 	</dependencies>
 
+	<distributionManagement>
```

Comment:
Please move either all pom.xml changes to the very first commit so we can drop it when merging or do not change the pom.xml besides version numbers at all.